base/Angle.hpp

```diff
@@ -120,7 +120,7 @@ class Angle
      */
     bool inline isApprox( Angle other, double prec = 1e-5 ) const
     {
-	return fabs( other.rad - rad ) < prec;
+        return Angle(fabs( other.rad - rad )).getRad() < prec;
```

Comment:
Angle(fabs(other-this)).getRad() can be negative if e.g. other-this is in [PI,2*PI]+. You want fabs(Angle(other.rad - rad).getRad()) (and you definitely will want to add a test for this particular corner case ...)